docs/resources.md

```diff
@@ -0,0 +1,97 @@
+# Zero components
+**s3 bucket: Terraform-state:** 
+`<project>-<env>-terraform-state`
+Terraform state s3 bucket is used for storing your terraform state in a terraform [remote backend][tf-remote-backend]
+
+**dynamo db: Terraform statelock**:
+Terraform uses [state lock][tf-remote-state] to ensure actions that could contradict each others do not happen at the same time 
+
+**IAM user: `<project>-ci-user`**:
+This user is given the ECR full-access policy, our intention is you will push your container images to ECR via this IAM
```

Comment:
This user also has access to assume the kubernetes-admin role, which will let it operate the Kuberenetes cluster.

Reply:
And has access to the s3 bucket we create for static assets.